api_examples/validate_receipt_smtp.js

```diff
@@ -0,0 +1,64 @@
+const request = require('superagent');
+const nodemailer = require('nodemailer');
+
+const checkReceived = (receiveAddress, sendAddress, baseURL, headers) => {
+    const apiURL = `${baseURL}/addresses/${receiveAddress}/messages`;
+    request
+        .get(apiURL)
+        .set(headers)
+        .then(res => {
+            res.body.forEach((message) => {
```

Comment:
If the intention is to stop running the loop after checking the message was received, you'll have to return something (like `const found = true/false`) from the .then() block, and return the promise chain starting at `request.get()`. In which case it may make more sense to convert the `res.bodylforEach` here to a normal `for` loop or something else where you can return the `found` status.